manifests/03-rbac-role.yaml

```diff
@@ -122,6 +122,14 @@ rules:
   - watch
   resourceNames:
   - console
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
```

Comment:
I'd combine this with the existing rule for api group `config.openshift.io`

Reply:
We don't want to `update`, `delete`, `create`. 
Just watch rules.

Reply:
Thanks, missed that. Looks good.